src/wallet/spend.cpp

```diff
@@ -348,6 +353,16 @@ CoinsResult AvailableCoins(const CWallet& wallet,
         }
     }
 
+    if (feerate.has_value()) {
+        std::map<COutPoint, CAmount> map_of_bump_fees = wallet.chain().CalculateIndividualBumpFees(outpoints, feerate.value());
```

Comment:
In 35a35fa989f14406f8218c20c4e4e732cee0bd6c: Shouldn't we ensure that `map_of_bump_fees` is not empty?

Reply:
I guess technically the `CalculateIndividualBumpFee(…)` call could return an empty map if the feerate changed during the call. I put an assert, and ran all the tests fine. In the worst case, it would fail right below where it applies the bump fees, so I’m not sure an assert would change much.

Reply:
Makes sense, just wondering about it.

Reply:
I see, thanks for asking!